cypress/plugins/tasks.js

```diff
@@ -84,27 +71,26 @@ const stackConfigTask = (_, config) => {
 const sqlTask = on => {
   on('task', {
     execSql: async sql => {
-      client = new Client(pgConfig)
+      const client = new Client(pgConfig)
       await client.connect()
       const res = await client.query(sql)
       client.end()
-
       return res
     },
-    dropAndSeedDatabase: () => {
-      postgresContainer.stdin.write(
-        `dropdb --if-exists --force ${devDatabase}; psql template1 -c "CREATE DATABASE ${devDatabase} TEMPLATE ${seedDatabase};"\n`,
-      )
-      return Promise.all([
-        new Promise(resolve => exec('tools/bin/mage dev:redisFlush', resolve)),
-        new Promise(resolve => {
-          postgresContainer.stdout.on('data', resolve)
-          postgresContainer.stderr.on('data', data => {
-            throw new Error(data)
-          })
-          postgresContainer.on('close', resolve)
-        }),
+    dropAndSeedDatabase: async () => {
+      const exec = util.promisify(childProcess.exec)
+      const res = await Promise.all([
+        exec('tools/bin/mage dev:sqlRestore'),
+        exec('tools/bin/mage dev:redisFlush'),
       ])
+      const err = res
+        .filter(e => !!e.stderr)
+        .map(e => e.stderr)
```

Comment:
```suggestion
        .map(e => e.stderr.toString())
```
The outs are byte arrays that need to be converted to strings.

Reply:
I've got them as `string`s actually. Where do you see this?

Reply:
Oh ok, seems like `exec` is already taking care of that then. Disregard.